From ae64629c5a568f228e5efba63ccd0e6378b8dc4c Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Fri, 13 Nov 2015 16:37:57 -0500
Subject: [PATCH 2/2] cryptodev: replace get_unused_fd to get_unused_fd_flags

get_unused_fd is deprecated and not valid anymore, replace it to
get_unused_fd_flags.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 drivers/staging/crypto/cryptodev/ioctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/crypto/cryptodev/ioctl.c b/drivers/staging/crypto/cryptodev/ioctl.c
index 5a55a76..b23f5fd 100644
--- a/drivers/staging/crypto/cryptodev/ioctl.c
+++ b/drivers/staging/crypto/cryptodev/ioctl.c
@@ -546,7 +546,7 @@ static int
 clonefd(struct file *filp)
 {
 	int ret;
-	ret = get_unused_fd();
+	ret = get_unused_fd_flags(0);
 	if (ret >= 0) {
 			get_file(filp);
 			fd_install(ret, filp);
-- 
2.0.2

